vaadin-virtual-list-flow-parent/vaadin-virtual-list-flow-integration-tests/src/test/java/com/vaadin/flow/component/virtuallist/tests/VirtualListIT.java

```diff
@@ -380,7 +380,8 @@ public void nativeButtonRenderer() {
     @Test
     public void numberRenderer() {
         List<TestBenchElement> items = $("vaadin-virtual-list")
-                .id("list-with-numbers").$("span").all();
+                .id("list-with-numbers")
+                .$("div[style*=\"position: absolute;\"]").all();
```

Comment:
Would `:scope > div` work here? 

Seems that the `:scope` selector is well-supported according to [caniuse](https://caniuse.com/?search=%3Ascope).

Reply:
Would be cleaner but for some reason with that selector the tests fail:

<img width="559" alt="Screenshot 2021-06-17 at 12 20 00" src="https://user-images.githubusercontent.com/1222264/122369517-ef19ee80-cf66-11eb-9c99-14812010a695.png">